app/controllers/host_aggregate_controller.rb

```diff
@@ -73,7 +74,7 @@ def delete_host_aggregates
       host_aggregate = HostAggregate.find(host_aggregate_id)
       if host_aggregate.nil?
         add_flash(_("Host Aggregate no longer exists."), :error)
-      elsif !host_aggregate.supports?(:delete_aggregate)
+      elsif !host_aggregate.supports?(:delete)
```

Comment:
not for this PR but planting a seed.
This looks like `unsupported_reason`
I do not think our localization for that is up to snuff, so when we pull out this concept, maybe we can fix the localization issues.

Reply:
also, thanks for the `s/delete_aggregate/delete/g`